bosh-director/lib/bosh/director/instance_updater/stopper.rb

```diff
@@ -48,9 +48,6 @@ def wait_for_dynamic_drain(initial_drain_time)
       drain_time = initial_drain_time
 
       loop do
-        # This could go on forever if drain script is broken, canceling the task is a way out.
-        @config.task_checkpoint
-
```

Comment:
@cppforlife thanks for raising it, found i forgot to include a change in 'instance_deleter' to my commit.